pages/index.tsx

```diff
@@ -27,20 +35,47 @@ const Home = ({ featuredTopicRef }: { featuredTopicRef: string }) => {
 
   const webpageTitle: string = homePageTranslation("webpageTitle")
   const heroHeading: string = homePageTranslation("heroHeading")
+  const heroSubheading: string = homePageTranslation("heroSubheading")
   const heroViewTopicsBtn: string = homePageTranslation("viewTopics")
   const mainBlurb: string = homePageTranslation("mainBlurb")
 
-  const whyJapanHeading: string = homePageTranslation("whyJapanHeading")
-  const whyJapanText: string = homePageTranslation("whyJapanText")
-  const prayerCTAHeading: string = homePageTranslation("prayerCTAHeading")
-  const prayerCTATextMap: string[] = homePageTranslation("prayerCTAText", { returnObjects: true })
+  const invitationTitle: string = homePageTranslation("invitationTitle")
+  const invitationBlurb: string = homePageTranslation("invitationBlurb")
+
+  const bibleVerse: string = homePageTranslation("bibleVerse")
+  const bibleRef: string = homePageTranslation("bibleRef")
+
+  const wePray: string = homePageTranslation("wePray")
+  const callToAction: string = homePageTranslation("callToAction")
+
+  // const whyJapanHeading: string = homePageTranslation("whyJapanHeading")
```

Comment:
remember to tidy up and clear out unused code before making the draft final (if applicable)